app/views/welcome/index.html.slim

```diff
@@ -16,7 +16,13 @@ section#badge
 
       form#get-badge-code
         .row.collapse
+          .small-12.columns
+            button#pick-email.button.active
+              | Email
+            button#pick-twitter.button
+              | Twitter
           .small-9.columns
+            input#badge-twitter.hide type="text" placeholder="@AwesomeCoder"
```

Comment:
Should remove `@` during submit event or don't suggest to the user that they should provide one. Twitter does not allow the `@` symbol as part of the DM message. For example the following is wrong:

```
DM @handle lorem ipsum
```